File: 백준/Bronze/10807. 개수 세기/개수 세기.cc
#include <stdio.h>

int main()
{
    int N = 0, V = 0, count = 0;
    scanf("%d", &N);
    
    int array[100];
    
    for(int i = 0; i < N; i++)
    {
        scanf("%d", &array[i]);
    }
    scanf("%d", &V);
    
    for(int i = 0; i < N; i++)
    {
        if(array[i] == V)
        {
            count++;
        }
    }
    printf("%d", count);
    return 0;
}

File: 백준/Bronze/10807. 개수 세기/개수 세기.java
import java.util.Scanner;

public class Main
{
    public static void main(String[] args)
    {
        Scanner scanner = new Scanner(System.in);
        int N = scanner.nextInt();
        int[] array = new int[100];  // 수정된 부분
        int count = 0;
        
        for(int i = 0; i < N; i++)
        {
            array[i] = scanner.nextInt();
        }
        
        int V = scanner.nextInt();
        
        for(int i = 0; i < N; i++)
        {
            if(array[i] == V)
            {
                count++;
            }
        }
        System.out.println(count);
        scanner.close();
    }
}